sdk/js/src/token_bridge/getIsTransferCompleted.ts

```diff
@@ -159,3 +159,25 @@ export async function getIsTransferCompletedAlgorand(
   const retVal: boolean = await checkBitsSet(client, appId, seqAddr, seq);
   return retVal;
 }
+
+/**
+ * <p>Returns true if this transfer was completed on Near</p>
+ * @param near account
+ * @param tokenAccount the Token bridge account
+ * @param signedVAA VAA to check
+ * @returns true if VAA has been redeemed, false otherwise
+ */
+export async function getIsTransferCompletedNear(
+  client: nearAccount,
+  tokenAccount: string,
+  signedVAA: Uint8Array
+): Promise<boolean> {
+  // Could we just pass in the vaa already as hex?
+  let vaa = Buffer.from(signedVAA).toString("hex");
+
+  return (
+    await client.viewFunction(tokenAccount, "is_transfer_completed", {
```

Comment:
is_transfer_completed returns an array of bools?

Reply:
Yes,   since you have to feed a vaa twice into the contract (two submits), it returns if it has been seen at all (first bool) and if it has been executed (second bool)